src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
+  } else {
+    return 80;
+  }
```

Comment:
The function uses `else` after `if` with `return`. This is a code style issue. It's better to avoid `else` after `if` with `return`, because the code after `if` with `return` won't be executed anyway. This makes the code cleaner and easier to read.